src/components/atoms/footer-main/footer-main.tsx

```diff
@@ -0,0 +1,142 @@
+import Image from 'next/image';
+import carbonNeutral from '../../../images/logos/carbon-neutral.svg';
+import styles from './footer-main.module.scss';
+
+/* eslint-disable max-len */
+
+export const FooterMain = (): JSX.Element => (
+  <footer className={styles['site-footer']}>
+
+    <div className={styles['site-footer__container']}>
```

Comment:
Should this be separate from `<footer>`? Does that make it (or the `<div>`) an empty element? I think it might be necessary for the top-border/divider, but I'm just sanity checking it?

Reply:
Footer deals with the border, padding and 100% width where as the container sets up the flex box and sets the max-width to 1296px so both are needed.